src/components/Modal.jsx

```diff
@@ -195,6 +248,8 @@ Modal.defaultProps = {
   title: 'Please give me a title :)',
   content: 'Please give me some content :)',
   image: '',
+  sound: '',
```

Comment:
Since this exists conditionally, I'd make this a boolean, and instead of having a `hasSound()` method, just check `if (this.props.sound) { // do stuff }`